1-triggering-jenkins.md

```diff
@@ -0,0 +1,115 @@
+# Triggering a Jenkins build from a push to GitHub
+
+Upload the app you want to test to a GitHub repo. 
+
+Go to Jenkins and create a new freestyle project.
+
+#### 1.1 Install the GitHub integration plugin
+
+* On Jenkins, go to 'Manage Jenkins' and then click on 'Manage Plugins'. 
+
+![manage-jenkins](assets/2-manage-jenkins.png)
+
+* Click on the tab 'Available', search for the GitHub Integration Plugin, and install it.
+
+![github-integration-plugin](assets/2-github-integration-plugin.png)
+
+#### 1.2 Configure GitHub repo to push to Jenkins
+
+* Go to your Github, click on Settings, and then go to 'Integration & services'. Add the Jenkins Github plugin.
+
+![jenkins-git](assets/2-jenkinsgit.png)
+
+* Once you've added the plugin, click on it. For 'Jenkins hook url', enter in the URL of your Jenkins instance, e.g. https://awesome-mapp.com/, followed by /github-webhook/ 
+
+![jenkins-hook](assets/2-jenkins-hook.png)
+
+**SSH Key**
+
+If you do not already have an SSH key, you can generate one with the following command in your terminal. 
+
+```
+ssh-keygen
+```
+Copy the public key. This may vary depending on where the key was generated. 
+```
+cat /var/lib/jenkins/.ssh/id_rsa.pub
+```
+In GitHub, go to 'Settings' and 'Deploy keys'. Click on 'Add deploy key' and paste the SSH key you just created. 
+
+![deploy-key](assets/2-deploy-key.png)
+
+Since we are running Jenkins in a Docker container, we need to run commands as the Jenkins user and not just in your local terminal. 
```

Comment:
We don't ask the client installing Jenkins by using Docker in the prerequisites. What should client do if they use brew or download directly to install Jenkins?